vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution.cc

```diff
@@ -91,7 +91,6 @@ void Contribution::OnProcessContributionQueue(
   }
 
   InitReconcile(std::move(info));
-  CheckContributionQueue();
```

Comment:
This is problematic part as we are checking queue immediately and not wait for old one to be deleted first